Update/haji_001.txt

```diff
@@ -22,16 +22,16 @@ void main()
 //魅音rvS23/03/VTH_mion1001.えー、この物語はフィクションです。krvS23/03/VTH_mion1002.間違いなくフィクションです。krvS23/03/VTH_mion1003.よって実在する場所、人物、団体名などとはr一切関係ありません。krvS23/03/VTH_mion1004.……よいこのみんな、真似しちゃダメだよ？
 	PlaySE(4, "ps3/s23/03/vth_mion1001", 256, 64);
 	OutputLine(NULL, "えー、この物語はフィクションです。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Eh, this story is a work of fiction.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/03/vth_mion1002", 256, 64);
 	OutputLine(NULL, "間違いなくフィクションです。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Without a doubt, it's fiction.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/03/vth_mion1003", 256, 64);
 	OutputLine(NULL, "よって実在する場所、人物、団体名などとはr一切関係ありません。",
-			NULL, "", Line_WaitForInput);
+			NULL, "Any resemblance to places, people, or corporations is purely coincidental.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/03/vth_mion1004", 256, 64);
 	OutputLine(NULL, "……よいこのみんな、真似しちゃダメだよ？",
-			NULL, "", Line_Normal);
+			NULL, "Plagiarism is a no-no, okay everyone?", Line_Normal);
```

Comment:
Are you sure this isn't supposed to be a "don't do this at home" / "don't actually do this"?